tests/multimodal/test_mapper.py

```diff
@@ -31,7 +31,6 @@ def test_clip_image_processor(image_assets, mm_registry, dtype, size_factor):
         dtype=dtype,
         revision=None,
```

Comment:
Need to set `limit_per_prompt` in `ModelConfig`.

Reply:
Do we? It's default to `None` and `MultiModalConfig`should initialize itself with default 1 in `_init_multimodal_config`, no?

Reply:
Since the default behaviour may change in the future (according to the `TODO`), it's best to explicitly specify this. Also there are a few tests that input multiple images.

Reply:
Oh sorry I missed those tests - updating!